### PR TITLE
add login/register pages

### PR DESCRIPTION
**Pull request by npelenskii**

### CODE REVIEW

**Review by ValeriaRbk — APPROVED** [review]

**Review by Bogd1y — APPROVED** [review]

File: src/redux/user/userActions.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  async ({ firstName, email, password }, { rejectWithValue }) => {
+    const config = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+
+    try {
+      await axios.post(
+        '/api/user/register',
```
Comment (ShevtsovVolodymyr) [inline src/redux/user/userActions.js:15]: Please consider using env vars or consts for endpoint urls.
```suggestion
        `{process.env.API_URL}/user/register`,
```

Comment (npelenskii) [reply src/redux/user/userActions.js:15]: I’m using proxy in package.json

Comment (npelenskii) [reply src/redux/user/userActions.js]: Changed, this will be better)

File: src/redux/user/userActions.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        return rejectWithValue(error.response.data.message);
+      } else {
+        return rejectWithValue(error.message);
+      }
+    }
+  }
+);
+
+export const loginUser = createAsyncThunk(
+  `user/login',
```
Comment (KMyroslav) [inline src/redux/user/userActions.js:30]: 🔧  Different quotes sumbols -> ` and '

Comment (npelenskii) [reply src/redux/user/userActions.js]: Fixed

File: src/redux/user/userActions.js (hunk 1 above)
Comment (KMyroslav) [reply src/redux/user/userActions.js]: We can also import axios in index.js and set default url to global axios object:

` _index.js_

import axios from "axios";
axios.defaults.baseURL = process.env.API_URL;`

**Review by KMyroslav — APPROVED** [review]

**Review by ShevtsovVolodymyr — APPROVED** [review]

**Pull request merged**